lib/log.js

```diff
@@ -1,18 +1,18 @@
 'use strict';
 
 var util = require('util');
-var clc = require('cli-color');
+var colors = require('picocolors');
 var figlet = require('figlet');
 var ansiRegex = require('ansi-regex');
 
 var xUtils = require('..');
 
 var colors = {
```

Comment:
`colors` is already used, I'm fixing ...

Reply:
Sorry, my bad.